### ISSUE TITLE
journalctl: document that --pager-end argument implies last boot and ways to override that

### ISSUE THREAD

**Issue by modax** [systemd/systemd#35248]
### systemd version the issue has been seen with

255

### Used distribution

Ubuntu 24.04 LTS

### Linux kernel version used

6.8.0-48-generic

### CPU architectures issue was seen on

x86_64

### Component

journalctl

### Expected behaviour you didn't see

`journalctl -e` to always display the last 1000 lines from the log as documented in the man page:

> Immediately jump to the end of the journal inside the implied pager tool. This implies -n1000 to guarantee that the pager will not buffer logs of unbounded size. This may be overridden with an explicit -n with some other numeric value, while -nall will disable this cap. Note that this option is only supported for the less(1) pager.
>
> Added in version 198.

### Unexpected behaviour you saw

`journalctl -e` [limits](https://github.com/systemd/systemd/blob/52b0351a154e9ad3f6a62c5de61198e454d86ecb/src/journal/journalctl.c#L459) the output to the last boot by default. This is not documented in the manual page anywhere.

Logs from all boots can be shown by specifying `-ball` **after** and only **after** `-e` option. I think this could be documented documented as well. If possible, it would be nice to have "after and only after" bit resolved to `-ball` anywhere since it is a bit confusing.

### Steps to reproduce the problem

`# journalctl -e -u dbus.service`
```
Nov 18 12:14:50 ubuntuserver systemd[1]: Starting dbus.service - D-Bus System Message Bus...
Nov 18 12:14:50 ubuntuserver dbus-daemon[688]: [system] AppArmor D-Bus mediation is enabled
Nov 18 12:14:50 ubuntuserver systemd[1]: Started dbus.service - D-Bus System Message Bus.
Nov 18 12:14:52 ubuntuserver dbus-daemon[688]: [system] Activating via systemd: service name='org.freedesktop.timedate1' unit='dbus-org.freedesktop.timedate1.service' requested by ':1.12' (uid=0 pid=702 comm="/usr/lib/snapd/snapd" label="unconfined")
Nov 18 12:14:52 ubuntuserver dbus-daemon[688]: [system] Successfully activated service 'org.freedesktop.timedate1'
Nov 18 13:03:46 ubuntuserver-noble-modax dbus-daemon[688]: [system] Activating via systemd: service name='org.freedesktop.fwupd' unit='fwupd.service' requested by ':1.79' (uid=990 pid=3965 comm="/usr/bin/fwupdmgr refresh" label="unconfined")
Nov 18 13:03:46 ubuntuserver dbus-daemon[688]: [system] Successfully activated service 'org.freedesktop.fwupd'
```
`# journalctl -ball -e -u dbus.service`
```
Nov 18 12:14:50 ubuntuserver systemd[1]: Starting dbus.service - D-Bus System Message Bus...
Nov 18 12:14:50 ubuntuserver dbus-daemon[688]: [system] AppArmor D-Bus mediation is enabled
Nov 18 12:14:50 ubuntuserver systemd[1]: Started dbus.service - D-Bus System Message Bus.
Nov 18 12:14:52 ubuntuserver dbus-daemon[688]: [system] Activating via systemd: service name='org.freedesktop.timedate1' unit='dbus-org.freedesktop.timedate1.service' requested by ':1.12' (uid=0 pid=702 comm="/usr/lib/snapd/snapd" label="unconfined")
Nov 18 12:14:52 ubuntuserver dbus-daemon[688]: [system] Successfully activated service 'org.freedesktop.timedate1'
Nov 18 13:03:46 ubuntuserver-noble-modax dbus-daemon[688]: [system] Activating via systemd: service name='org.freedesktop.fwupd' unit='fwupd.service' requested by ':1.79' (uid=990 pid=3965 comm="/usr/bin/fwupdmgr refresh" label="unconfined")
Nov 18 13:03:46 ubuntuserver dbus-daemon[688]: [system] Successfully activated service 'org.freedesktop.fwupd'
```
`# journalctl -e -ball -u dbus.service`
```
Nov 18 11:08:04 ubuntuserver dbus-daemon[687]: [system] Activating via systemd: service name='org.freedesktop.PackageKit' unit='packagekit.service' requested by ':1.559' (uid=0 pid=29069 comm="/usr/bin/gdbus call --system --dest org.freedeskto" label="unconfined")
Nov 18 11:08:04 ubuntuserver dbus-daemon[687]: [system] Successfully activated service 'org.freedesktop.PackageKit'
Nov 18 11:14:03 ubuntuserver-noble-modax dbus-daemon[687]: [system] Activating via systemd: service name='org.freedesktop.fwupd' unit='fwupd.service' requested by ':1.565' (uid=990 pid=29252 comm="/usr/bin/fwupdmgr refresh" label="unconfined")
Nov 18 11:14:03 ubuntuserver dbus-daemon[687]: [system] Successfully activated service 'org.freedesktop.fwupd'
Nov 18 12:14:33 ubuntuserver dbus-daemon[687]: [system] Activating via systemd: service name='org.freedesktop.fwupd' unit='fwupd.service' requested by ':1.572' (uid=990 pid=29761 comm="/usr/bin/fwupdmgr refresh" label="unconfined")
Nov 18 12:14:34 ubuntuserver dbus-daemon[687]: [system] Successfully activated service 'org.freedesktop.fwupd'
-- Boot db9461d4210445528dee9ff3407c5407 --
Nov 18 12:14:50 ubuntuserver systemd[1]: Starting dbus.service - D-Bus System Message Bus...
Nov 18 12:14:50 ubuntuserver dbus-daemon[688]: [system] AppArmor D-Bus mediation is enabled
Nov 18 12:14:50 ubuntuserver systemd[1]: Started dbus.service - D-Bus System Message Bus.
Nov 18 12:14:52 ubuntuserver dbus-daemon[688]: [system] Activating via systemd: service name='org.freedesktop.timedate1' unit='dbus-org.freedesktop.timedate1.service' requested by ':1.12' (uid=0 pid=702 comm="/usr/lib/snapd/snapd" label="unconfined")
Nov 18 12:14:52 ubuntuserver dbus-daemon[688]: [system] Successfully activated service 'org.freedesktop.timedate1'
Nov 18 13:03:46 ubuntuserver dbus-daemon[688]: [system] Activating via systemd: service name='org.freedesktop.fwupd' unit='fwupd.service' requested by ':1.79' (uid=990 pid=3965 comm="/usr/bin/fwupdmgr refresh" label="unconfined")
Nov 18 13:03:46 ubuntuserver-noble-modax dbus-daemon[688]: [system] Successfully activated service 'org.freedesktop.fwupd'
```

**Comment by yuwata** [comment]
Note, -e is slightly (heavily?) slow if journal file is huge. I recommend to use -r if it is OK that the ordering is inverted. I always use `-r` rather than `-e`.

**Comment by modax** [comment]
Nice! Thanks for the fix and tip!
